browser/src/App.js

```diff
@@ -58,19 +87,28 @@ function App() {
         Nightfall Phase2 Ceremony
       </div>
       <p>
-        Zero-knowledge proofs require a trusted setup. Since Nightfall uses the Groth16 proving
-        scheme, a second phase of the MPC is needed, for each circuit. We want to invite you to
-        contribute to this Second Phase. To start, just move your {isMobile ? 'device' : 'cursor'}{' '}
-        to generate some entropy. If you want, you can also enter your name for later verification.
-      </p>
-      <p>The process takes 10-20mins. Go grab a coffee!</p>
-      <ContributeCard
-        setEntropy={setEntropy}
-        entropy={entropy}
-        entropyArr={entropyArr}
-        circuits={CIRCUITS}
-        isMobile={isMobile}
-      />
+          Zero-knowledge proofs require a trusted setup. Since Nightfall uses the Groth16 proving
+          scheme, a second phase of the MPC is needed, for each circuit. We want to invite you to
+          contribute to this Second Phase. To start, just move your {isMobile ? 'device' : 'cursor'}{' '}
+          to generate some entropy. If you want, you can also enter your name for later verification.
+        </p>
+      <div style={{display: token == '' ? 'none' : 'inline'}}>
```

Comment:
Instead of manipulating display property, it is better not to render it at all. Like so…

```jsx
{ token !== '' ? <p>The process might…</p> : null }
```

Reply:
The problem is that doing this inline with such a "big" div tag might not make the code so clear.

Reply:
Up to you, but I don't think that ternary like that makes it unclear. It is defacto conditional rendering method in React ecosystem. Manipulating CSS display feels hacky, and semantically it makes more sense for this element not to be rendered at all.